package.json

```diff
@@ -4,6 +4,7 @@
   "description": "![GitHub Last Commit](https://img.shields.io/github/last-commit/samrogers15/duplicate-reconciliation-program?style=plastic) ![GitHub Repo Size](https://img.shields.io/github/repo-size/samrogers15/duplicate-reconciliation-program?style=plastic) ![GitHub Followers](https://img.shields.io/github/followers/samrogers15?style=social) ![GitHub](https://img.shields.io/github/languages/top/samrogers15/duplicate-reconciliation-program?style=plastic)",
   "main": "deduplicator.js",
   "scripts": {
+    "start": "node ./main.js",
```

Comment:
Aliasing a simple command to run the project is a much nicer user experience than having to write something specific in the terminal each time you want to run the program. This ties into to the comment I wrote about setting up a clear entry point into your app: if you have that established, it's very straightforward to set up a command to start the program.

Reply:
On a similar note, I saw in your `README` that you have instructions for how to swap the import statements to evaluate the larger dataset, which is pretty clunky for the reviewer. With this change in place, it'd be trivial to add a second command (or a flag in this command) that could be intercepted by some logic in `main.js` and then run the primary `output` function logic with the desired dataset depending on which command the user ran. Much nicer and then they don't have to do any extra work or interact with the code you wrote -- they run everything they need to see via CLI commands.

Reply:
node process.argv for a if/else to select actual file.